packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -42,4 +65,13 @@ export class DbSyncNetworkInfoProvider extends DbSyncProvider implements Network
       totalSupply
     });
   }
+
+  async start(): Promise<void> {
+    this.#pollId = pollDbSync(this.#cache, () => this.#builder.queryLatestEpoch(), this.#dbPollInterval);
```

Comment:
This would be better, wouldn't leak implementation details (setInterval):
```suggestion
    this.#pollService = pollDbSync(this.#cache, () => this.#builder.queryLatestEpoch(), this.#dbPollInterval);
    ...
    async close(): Promise<void> {
      this.#pollService?.shutdown();
      this.#cache.shutdown();
    }
```